src/main/scala/com/typesafe/sbt/packager/docker/DockerSpotifyClientPlugin.scala

```diff
@@ -0,0 +1,83 @@
+package com.typesafe.sbt
+package packager
+package docker
+
+import java.nio.file.Paths
+
+import com.spotify.docker.client.messages.ProgressMessage
+import com.spotify.docker.client.{ProgressHandler, DockerClient, DefaultDockerClient}
+import com.spotify.docker.client.DockerClient.BuildParameter._
+import sbt._
+import sbt.Keys._
+import packager.Keys._
+import universal.UniversalPlugin.autoImport.stage
+
+
+/**
+ * == DockerSpotifyClientPlugin Plugin ==
+ *
+ * This plugin helps you build docker containers using Spotify Docker Client.
+ *
+ * == Configuration ==
+ *
+ * In order to configure this plugin take a look at the available [[com.typesafe.sbt.packager.docker.DockerKeys]]
+ *
+ * == Requirements ==
+ *
+ * You need docker to have docker installed on your system.
```

Comment:
this is actually not a requirement, via an [env var](https://github.com/spotify/docker-client/blob/b3b7371/src/main/java/com/spotify/docker/client/DefaultDockerClient.java#L1203) you can just point at a remote endpoint